src/badge/badge.js

```diff
@@ -35,9 +35,9 @@ const Badge = props => {
   let Component = View;
 
   if (component) {
-    Component = component
+    Component = component;
   } else if (onPress) {
-    Component = TouchableOpacity
+    Component = TouchableOpacity;
```

Comment:
Just curious, is it your own prettier or the pre-commit hook which added these ?

Reply:
Must be the pre-commit, I didn't step into those file.